asv/__init__.py

```diff
@@ -29,4 +29,4 @@
     if os.environ.get('__PYVENV_LAUNCHER__'):
         os.unsetenv('__PYVENV_LAUNCHER__')
 
-from . import plugin_manager
+    from . import plugin_manager   # noqa: F401 this updates command args to plugin args
```

Comment:
This indentation doesn't make sense.

For the comment, maybe better `unused, but required to load plugins`